drivers/flash/soc_flash_cc23x0.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) 2024 Texas Instruments Incorporated
+ * Copyright (c) 2024 BayLibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/flash.h>
+#include <zephyr/irq.h>
+#include <zephyr/kernel.h>
+#include <string.h>
+
+#include <driverlib/flash.h>
+#include <driverlib/vims.h>
+
+#define DT_DRV_COMPAT     ti_cc23x0_flash_controller
+#define SOC_NV_FLASH_NODE DT_INST(0, soc_nv_flash)
+
+#define FLASH_ADDR       DT_REG_ADDR(SOC_NV_FLASH_NODE)
+#define FLASH_SIZE       DT_REG_SIZE(SOC_NV_FLASH_NODE)
+#define FLASH_ERASE_SIZE DT_PROP(SOC_NV_FLASH_NODE, erase_block_size)
+#define FLASH_WRITE_SIZE DT_PROP(SOC_NV_FLASH_NODE, write_block_size)
+
+struct flash_priv {
```

Comment:
it'd be somewhat conventional to call this struct flash_cc23x0_data

Reply:
Fixed